code/workspaces/web-app/src/containers/adminListUsers/UserResources.js

```diff
@@ -117,3 +126,17 @@ export default function UserResources({ user, filters, roles }) {
     </div>
   );
 }
+
+const SystemRoleSkeleton = () => {
```

Comment:
Could use GridSkeleton

Reply:
I've replaced this with `GridSkeleton` as suggested